<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifierEmploi</class>
 <widget class="QDialog" name="ModifierEmploi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1530</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date d'Expiration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de Publication</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>1351</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Description</string>
   </property>
   <widget class="QTextEdit" name="textEdit_Description_emploi_mod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1311</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QDateEdit" name="dateEdit_Expiration_emploi_mod">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>240</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_Publication_Emploi_mod">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmModOffre">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>840</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_titreEmploi_mod">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_entreprise_Mod">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>124</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Entreprise_Mod">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>100</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Entreprise</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
